include/bout/invert/laplacexy.hxx

```diff
@@ -129,20 +142,26 @@ private:
   PC pc;            ///< Preconditioner
 
   Mesh *localmesh;   ///< The mesh this operates on, provides metrics and communication
+
+  static int instance_count;
+  int my_id = 0;
```

Comment:
This is to uniquely identify different instances of this type, right? Another option could be a public `std::string name`, and leave it to the user to identify? Or take the default value from the `Options` used to construct?

Just ideas for future polish, might be something we want to do more generally